.circleci/config.yml

```
@@ -18,7 +18,9 @@ cache-key: &cache-key
 commands:
   use_npm_token:
     steps:
-      - run: echo "//registry.npmjs.org/:_authToken=${NPM_TOKEN}" >> ~/.npmrc
+      - run: |
+      echo "//registry.npmjs.org/:_authToken=${NPM_TOKEN}" >> ~/.npmrc
```

Something is not right here, both indentation and registry address. Should be this?
```suggestion
        echo "//npm.pkg.github.com/:_authToken=${GITHUB_PACKAGES_READ_TOKEN}" > ~/.npmrc
```

---

Yep silly me

---

Indentation is still not there 🙂